src/modules/xworkspaces.cpp

```diff
@@ -201,6 +217,9 @@ namespace modules {
       }
       offset += step;
     }
+
+    m_desktop_occupied.resize(m_desktop_names.size(), 0);
```

Comment:
Is is correct to initialize new elements with `0`. My understanding is that it should be `false`, since the vector is declared as `vector<bool> m_desktop_occupied{};`. 

```suggestion
    m_desktop_occupied.resize(m_desktop_names.size(), false);
```

I might be wrong about this, though. The content of this tally-object change during implementation of the initial PR quite a bit back and forth, so there might be other areas that need adaptation.

Reply:
Yeah that's correct it's just a rest of a cancelled modition.